digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala

```diff
@@ -77,29 +74,32 @@ object DataFixture {
     }
   }
 
-  def importRoadAddresses(isDevDatabase: Boolean): Unit = {
+
+  def importRoadAddresses(isDevDatabase: Boolean, importTableName: Option[String]): Unit = {
     println(s"\nCommencing road address import from conversion at time: ${DateTime.now()}")
     val vvhClient = new VVHClient(dr2properties.getProperty("digiroad2.VVHRestApiEndPoint"))
     val geometryAdjustedTimeStamp = dr2properties.getProperty("digiroad2.viite.importTimeStamp", "")
     if (geometryAdjustedTimeStamp == "" || geometryAdjustedTimeStamp.toLong == 0L) {
       println(s"****** Missing or bad value for digiroad2.viite.importTimeStamp in properties: '$geometryAdjustedTimeStamp' ******")
     } else {
       println(s"****** Road address geometry timestamp is $geometryAdjustedTimeStamp ******")
-      val conversionTable = if (isDevDatabase) {
-        konvTableNameDev
-      } else {
-        konvTableNameProd
-      }
-      val importOptions = ImportOptions(
-        onlyComplementaryLinks = false,
-        useFrozenLinkService = dr2properties.getProperty("digiroad2.VVHRoadlink.frozen", "false").toBoolean,
-        geometryAdjustedTimeStamp.toLong, conversionTable,
-        onlyCurrentRoads = dr2properties.getProperty("digiroad2.importOnlyCurrent", "false").toBoolean)
-      dataImporter.importRoadAddressData(Conversion.database(), vvhClient, importOptions)
+      importTableName match {
+        case None => // shouldn't get here because args size test
+          println(s"****** Import failed! conversiontable name required as second input ******")
+          System.exit(-1)
```

Comment:
Can we avoid this System.exit(-1)? If we just return nothing I think the result will be the same

Reply:
k, thought it wouldn't show as error in jenkins if it didn't throw -1. sure i can remove it

Reply:
I think that it is not wrong at all, I just think we can handle it in a different way :) 
Like using the break() function or returning an exception.

Reply:
ok, thanks i will add that =)